ImageClassification/ViewControllers/PredictionAssist.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  PredictionAssist.swift
+//  ImageClassification
+//
+//  Created by Likhon Gomes on 4/11/20.
+//  Copyright © 2020 Y Media Labs. All rights reserved.
+//
+
+import Foundation
+
+class PredictionAssit {
+    
+    init(){}
+    
+    func similarity(){
+        
+        let alphabets = [["A","S","E","M","N","T"],["G","H"],["I","J","I"],["K","P"],["U","V"]]
```

Comment:
Was just glancing at this. It looks like you have I in the same list twice. Is this by design?